modules/shared/partials/acid-transactions.adoc

```diff
@@ -50,19 +43,30 @@ If durability is set to `None`, then atomicity _cannot be guaranteed_.
 // tag::creating[]
 == Creating a Transaction
 
-A core idea of the library is that the application supplies the logic for transaction inside a lambda, including any conditional logic required, and the transactions library takes care of getting the transaction committed.
+A core idea of the library is that the application supplies the logic for transaction inside a lambda, including any conditional logic required, and the transactions library takes care of getting the transaction committed.  If the library encounters a transient error, such as a temporary conflict with another transaction, then it can rollback what has been done so far and run the lambda again.
```

Comment:
might be more future proof to say "implementation" or something instead of "the library"?  What happens when parts aren't in a library?

Reply:
Well, we can change the docs if that happens :) The consistent use of "transactions library" is something I'm kind of debating with Shivani too.  I like it as I think we've created a confusing bunch of docs, blogs and I suspect upcoming marketing for our users that will have them puzzled by whether we have two forms of non-compatible transactions that seem to have completely different ways of initiating and "semantics" (sorry).  While I seem to be losing the fight to clearly call out the distinction in the server docs, I'm trying to ensure that this little piece I own makes it clearer.  So I'm explicitly using "transactions library" rather than a more generic "implementation" (or similar) so users know what they get with the lib and don't necessarily get for free with a BEGIN TRANSACTION.